libs/components/src/next/search-filter/vl-search-filter.css.ts

```diff
@@ -0,0 +1,63 @@
+import { css, CSSResult } from 'lit';
+
+export const searchFilterStyles: CSSResult = css`
+    :host {
+        --vl-form-label-font-weight: normal;
+        --vl-title-h2-font-size: 1.6rem;
+        --vl-title-h2-font-size-small: 1.4rem;
+        --vl-title-letter-spacing: 0.1rem;
```

Comment:
Die letter-spacing, waarom zouden we die globaal definieren (zonder ze daar dan te definieren maar op initial te laten) als die nergens nodig was tot nu toe behalve nu hier?